modules/integration_azure-functions/conf/readme.yaml

```diff
@@ -1,8 +1,6 @@
 documentations:
-  - name: Azure Monitor metrics
-    url: 'https://docs.microsoft.com/en-us/azure/azure-monitor/platform/metrics-supported'
```

Comment:
https://docs.microsoft.com/en-us/azure/azure-monitor/essentials/metrics-supported#microsoftwebsites

Reply:
With dimension "is Azure Function : true"

Reply:
oh good catch, I removed the doc because I only opended the `detectors-gen.tf` file and understood the metric does not come from azure integration so I supposed that the used metrics in module only come from signalfx serverless wrapper.

I missed the "manually" written detectors file which indeed use azure integrations metrics.